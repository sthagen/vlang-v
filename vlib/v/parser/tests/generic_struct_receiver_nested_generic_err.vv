struct Foo[T] {
	bar T
}

fn (a Foo[Foo[T]]) baz() {}
