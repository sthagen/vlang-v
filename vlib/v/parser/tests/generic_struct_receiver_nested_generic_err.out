vlib/v/parser/tests/generic_struct_receiver_nested_generic_err.vv:5:11: error: the parameter type name of a generic struct, must be a single capital letter placeholder name, like T or X, or a non-generic type name like int, string, etc.
    3 | }
    4 | 
    5 | fn (a Foo<Foo<T>>) baz() {}
      |           ~~~~~~
